exporter/elasticsearchexporter/README.md

```diff
@@ -25,19 +25,19 @@ This exporter supports sending OpenTelemetry logs to [Elasticsearch](https://www
 - `num_workers` (optional): Number of workers publishing bulk requests concurrently.
 - `index`: The
   [index](https://www.elastic.co/guide/en/elasticsearch/reference/current/indices.html)
-  or [datastream](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html)
-  name to publish events to. The default value is `logs-generic-default`. Note: To better differentiate between log indexes and traces indexes, `index` option are deprecated and replaced with below `logs_index`
+  or [data stream](https://www.elastic.co/guide/en/elasticsearch/reference/current/data-streams.html)
+  name to publish events to. The default value is `logs-generic-default`. Note: To better differentiate between log indices and traces indices, `index` option is deprecated and replaced with below `logs_index`
```

Comment:
```suggestion
  name to publish events to. The default value is `logs-generic-default`.
```
Nit: It makes sense to me to make the deprecation more obvious, so I added comments that make sense to me. Feel to ignore, reword, or reformat as you see fit.